app/components/tipper/TipPage/ShareUnclaimedTip.tsx

```diff
@@ -31,16 +31,18 @@ export function ShareUnclaimedTip({ tip }: ShareUnclaimedTipProps) {
     <>
       <Card css={{ dropShadow: "$sm" }}>
         <Card.Header>
-          <Row justify="center">
+          <Row justify="center" align="center">
             <Text size={20} b>
-              👇 Have your tippee scan this QR code
+              👇 Scan this QR code
```

Comment:
Should we make this change? I think we need it to be obvious that we want the recipient to scan, not the person who just created the tip?